src/net/sourceforge/kolmafia/request/AlliedRadioRequest.java

```diff
@@ -105,9 +105,7 @@ public static void postChoice(final String responseText, final boolean handheld,
     matcher = AlliedRadioRequest.GREY_TEXT_PATTERN.matcher(responseText);
     if (matcher.find()) {
       String text = matcher.group(1);
-      String message = "Radio grey text received: " + text;
```

Comment:
can you add logging the text back to the session log, please?

even if it's stored in a pref, prefs can be lost, so it's nice to have in the log too.